gating/periodic/run_system_tests.sh

```diff
@@ -29,7 +31,7 @@ fi
 # 1. Clone test repository into working directory.
 pushd "${SYS_WORKING_DIR}"
 git clone "${SYS_TEST_SOURCE_REPO}"
-cd "${SYS_TEST_SOURCE}"
+pushd "${SYS_TEST_SOURCE}"
```

Comment:
Just inquiring about the reason for `pushd` as i don't see any use of a corresponding `popd`.

Reply:
Brain fart on my part. Thanks for pointing out the lack of a `popd`. I have reverted this change since it is unnecessary.
Thanks @coreywright